bin/minify.js

```diff
@@ -12,23 +12,31 @@ const os = require( 'os' );
 /**
  * Internal dependencies
  */
-const files = fs
-	.readdirSync( './public' )
+
+const walkDir = ( dir ) => fs
```

Comment:
whats the reason for this change?

Reply:
Since we don't bundle the moment-locale files into the vendor chunk, we need to load them from the `public/moment/locale` directory. As the minifier only processed files in the `public/` directory, I did this changes, to also minify files in all child subdirectories.

Reply:
Thank you for the helpful explanation! Totally makes sense ⭐️